hazelcast/src/main/java/com/hazelcast/client/impl/ClientEngineImpl.java

```diff
@@ -347,11 +359,22 @@ public void shutdown(boolean terminate) {
         ownershipMappings.clear();
     }
 
+    public boolean checkAndSetGreaterThanLastAuthenticationCorrelationId(String clientUuid, long newCorrelationId) {
```

Comment:
minor: `setIfGreaterThanLastAuthenticationCorrelationId(...)`, to make naming symmetric.